src/main/java/roomescape/presentation/controller/auth/LoginController.java

```diff
@@ -0,0 +1,45 @@
+package roomescape.presentation.controller.auth;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.application.auth.AuthService;
+import roomescape.application.auth.dto.LoginRequest;
+import roomescape.application.auth.dto.LoginResponse;
+import roomescape.application.auth.dto.MemberIdDto;
+import roomescape.application.dto.MemberDto;
+import roomescape.infrastructure.AuthenticatedMemberId;
+
+@RestController
+@RequestMapping("/login")
+public class LoginController {
+
+    private final AuthService authService;
+
+    public LoginController(AuthService authService) {
+        this.authService = authService;
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> login(@Valid @RequestBody LoginRequest loginRequest, HttpServletResponse response) {
+        Cookie cookie = authService.createCookie(loginRequest);
+
+        response.addCookie(cookie);
+        response.setHeader("Set-Cookie", cookie.getAttribute("token"));
```

Comment:
`addCookie`와 `setHeader`는 동일한 작업입니다. 
헤더를 직접 설정하는 코드는 제거해주세요.

```suggestion
        response.addCookie(cookie);
```

Reply:
`addCookie()`는 자바 서블릿 API에서 제공하는 정식 쿠키 설정 방법으로 내부적으로 `Set-Cookie` HTTP 헤더를 만들어주고, `setHeader("Set-Cookie", "...")`는 직접 HTTP 헤더에 문자열로 쿠키를 설정하는 방식으로 동일한 작업이라는 점을 인지하지 못했던 것 같습니다.

`setHeader("Set-Cookie", "...")` 방식은 쿠키 포맷을 직접 문자열로 조립해야 하고, 이전에 설정된 헤더를 덮어쓰기 때문에 실수할 여지가 크므로 `addCookie()`를 사용하는 것이 좋다는 것을 배웠습니다.
따라서 말씀해준대로 헤더를 직접 설정하는 코드를 삭제하였습니다.